source/install/install-ubuntu-1604-postgresql.rst

```diff
@@ -100,5 +100,9 @@ Assume that the IP address of this server is 10.10.10.1
       You might have to install the PostgreSQL client software to use the command.
 
   The PostgreSQL interactive terminal starts. To exit the PostgreSQL interactive terminal, type ``\q`` and press **Enter**.
+  
+13. (Optional) If you don't want to enter your password at each login, you can use a ``.pgpass`` file . It is a special file which has to be assigned ``0600`` in permissions otherwise it will not work. One can also pass the location of the file using the ``PGPASSFILE`` env variable.
+
+This leaves the security to the user running the application. If you run it as a service, then you need *sudo* permissions to start a service file. As a result only *root* user has read-write access. 
```

Comment:
Maybe add a link to the official postgres docs like so for more info on the pgpass file as it's not mattermost specific and some details might change in the future. Other than that it looks good to me.

```suggestion
This leaves the security to the user running the application. If you run it as a service, then you need *sudo* permissions to start a service file. As a result only *root* user has read-write access. 

For further info on the pgpass file please refer to `the official PostgreSQL documentation<https://www.postgresql.org/docs/9.4/libpq-pgpass.html>`_.
```

Reply:
@svelle this is a nice suggestion, will commit the changes

Reply:
Perfect, thanks.
Now if @justinegeffen doesn't have any issues with wording this should be good to merge :)